### DISCUSSION TITLE
Tooltip is not hiding, if we disabled the button on click

### DISCUSSION THREAD

**Discussion by jitangupta** [floating-ui/floating-ui#1266] [Q&A]
```           
 $("#btn").on("click", function () {
                $(this).prop("disabled", true);
            });
```
here is an running example. https://jsfiddle.net/JitanGupta/e8fjd0nL/

I am using basic setup with bootstrap 4

Off course, we can do `$('[data-toggle="tooltip"]').tooltip('hide');` hide the tooltip, but still solution is not justifiable to me.

Any suggestion would help.
Thanks

**Comment by FezVrasta** [comment]
Hi, this repository is about Popper, not Bootstrap.
